src/main/java/co/rsk/federate/btcreleaseclient/BtcReleaseClient.java

```diff
@@ -357,31 +364,32 @@ protected void validateTxCanBeSigned(BtcTransaction btcTx) throws FederatorAlrea
                 if (spendingFedFilter.isEmpty()) {
                     String message = String.format(
                             "Transaction %s can't be signed by any of the observed federations",
-                            btcTx.getHash()
+                            pegoutBtcTx.getHash()
                     );
                     throw new FederationCantSignException(message);
                 }
             }
         } catch (SignerException e) {
             String message = String.format("[validateTxCanBeSigned] Error validating tx %s, " +
-                    "failed to get current federator public key", btcTx.getHashAsString());
+                    "failed to get current federator public key", pegoutBtcTx.getHashAsString());
             logger.error(message, e);
         }
     }
 
-    protected void signRelease(int signerVersion, ReleaseCreationInformation releaseCreationInformation) {
+    protected void signRelease(int signerVersion, ReleaseCreationInformation pegoutCreationInformation) {
+        final String topic = "btcrelease";
         try {
-            logger.debug("[signRelease] HSM signer version {}", signerVersion);
-            logger.debug("[signRelease] Going to sign tx {}", releaseCreationInformation.getInformingRskTxHash());
-            logger.trace("[signRelease] Enforce signer requirements");
-            releaseRequirementsEnforcer.enforce(signerVersion, releaseCreationInformation);
+            logger.debug("[signConfirmedPegout] HSM signer version {}", signerVersion);
```

Comment:
Did the method name change?

Reply:
It didn't. It came from the previous PR. Just fixed it.